walkthroughs/howto-k8s-cloudmap/README.md

```diff
@@ -33,9 +33,9 @@ You can use v1beta1 example manifest with [aws-app-mesh-controller-for-k8s](http
     ```
     export AWS_DEFAULT_REGION=us-west-2
     ```
-4. **ENVOY_IMAGE** environment variable is set to App Mesh Envoy, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+4. **(Optional) Specify Envoy Image version** If you'd like to use a different Envoy image version than the [default](https://github.com/aws/eks-charts/tree/master/stable/appmesh-controller#configuration), run `helm upgrade` to override the `sidecar.image.repository` and `sidecar.image.tag` fields, e.g.
     ```
-    export ENVOY_IMAGE=...
+    helm upgrade -i appmesh-controller eks/appmesh-controller --namespace appmesh-system --set sidecar.image.repository=840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-envoy --set sidecar.image.tag=<VERSION>
```

Comment:
This assumes the customer is using the helm flags specified in the command above. This may break for customer who has different flag set. I'd suggest pointing to the [eks-charts configuration instead](https://github.com/aws/eks-charts/tree/master/stable/appmesh-controller#configuration) and they can use the  `sidecar.image.repository` and `sidecar.image.tag` tag instead.

Alternatively, you can remove the need to specify the Envoy version all together

Reply:
removed the example commands.